Directory.Build.Props

```diff
@@ -46,6 +46,7 @@
       <SDLCheck>true</SDLCheck>
       <ConformanceMode>true</ConformanceMode>
       <LanguageStandard Condition="'$(CppWinRTLanguageStandard)'==''">stdcpp17</LanguageStandard>
+      <LanguageStandard Condition="'$(CppWinRTLanguageStandard)'=='stdcpp20'">stdcpp20</LanguageStandard>
```

Comment:
It seems inconsistent that we're using `latest` to mean the latest standard but `stdcpp20` to mean 20.

```suggestion
      <LanguageStandard Condition="'$(CppWinRTLanguageStandard)'=='20'">stdcpp20</LanguageStandard>
```

Reply:
Fixed in #1187